### ISSUE TITLE
Landing page "Mautic tracking is not initiated correctly"

### ISSUE THREAD

**Issue by lutek** [mautic/mautic#2902]
What type of report is this:

| Q  | A
| ---| ---
| Bug report? | Y
| Feature request? | N
| Enhancement? | N

## Description:
Page visit isn't tracked on landing pages, campaign isn't triggered to move contact to confirmed list.

![mautic_landing_pages](https://cloud.githubusercontent.com/assets/846598/20040810/787d4ef8-a45e-11e6-8d37-ee4facaa4ed7.jpeg)

the upper part of the html:
```
<html><head><title>Subskrypcja potwierdzenie</title><meta name="description" content="" /><link rel="stylesheet" href="/themes/oxygen/css/oxygen.css" type="text/css" /><script src="http://ma.DOMAIN.pl/mtc.js" async data-source="mautic"></script>

</head><body style="min-height: 300px;">
```

## If a bug:

| Q   | A
| --- | ---
| Mautic version | 2.2.1
| PHP version | 5.6.27-1~dotdeb+7.1

### Steps to reproduce:
1. Create campaign based on: https://www.mautic.org/blog/integrator/creating-a-double-opt-in-email-campaign/
2. Create Focus item with newsletter form.
3. Submit form.
4. Click on confirmation link, and visit landing page.
 
### Log errors: 
In browser:
Mautic tracking is not initiated correctly. Follow the documentation. mtc.js:11

**Comment by lutek** [comment]
the same problem on mautic.net
test page: https://fabrykawafelkow.mautic.net/subskrypcja-potwierdzenie

**Comment by dkaragiozov** [comment]
I can reproduce this bug on 2.3.0 version. Here is the test landing page:
https://mautic.tetracom.com/grzdefaultlanding

In the browse console I see two messages:
Mautic tracking is not initiated correctly. Follow the documentation.    mtc.js:13
Mautic tracking is not initiated correctly. Follow the documentation.    mtc.js:11

**Comment by scottbuscemi** [comment]
This bug still exists. Is there a workaround?

**Comment by scottbuscemi** [comment]
For the landing pages, it appears that the code is looking for the tracking pixel and throwing an error if it's not found, even if the tracking is being accomplished by javascript instead.

The mtc.js file needs to be updated so that it doesn't look for a pixel if the javascript is accomplishing the tracking.

I'm seeing the error "Mautic tracking is not initiated correctly", but the visit count is increasing as expected.

**Comment by escopecz** [comment]
I think this was fixed in Mautic 2.6.0. Could you check if it's fixed?

**Comment by lutek** [comment]
Checked on version 2.8.x it works.
